governance/votes/Executive vote - April 26, 2021.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Template - [Executive Vote] Approve April 2021 Governance Cycle - April 26, 2021
+
+---
+# [Executive Proposal] Approve April 2021 Governance Cycle - April 26, 2021
+
+The Governance Facilitator(s) and the MakerDAO Smart Contracts Domain Team have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this Executive proposal passes the following **MIP status changes** will occur:
+
+### MIP Amendments and Declaration of Intent
+* Approve MIP4c2-SP7: MIP4 Amendments
+* Approve MIP4c2-SP8: MIP 9 Amendments
+* Approve MIP4c2-SP13: MIP0 Amendments
+* MIP13c3-SP10: Declaration of Intent - eurDai will move to Accepted Status
+
+### Content Production Core Unit
```

Comment:
I would maybe summarize these as:

### Core Unit Onboarding
* GRO-001, Growth Core Unit Proposals will move to Accepted status.
* MDS-001, MakerDAO Shop Core Unit Proposals will move to Accepted status.
etc...

Reply:
Then move the budget details to the proposal detail section.

Reply:
Hmm, actually, maybe keep the budgets that are being approved here. That feels pretty critical.

Reply:
Maybe headers for: 'Core Unit Onboarding' , 'Budget Approvals' , 'Budget Payouts'?